src/REopt.jl

```diff
@@ -72,10 +73,10 @@ const FUEL_DEFAULTS = Dict(
         "diesel_oil"=>0.0
     ),
     "emissions_factor_lb_CO2_per_mmbtu" => Dict(
-        "natural_gas"=>116.9,
-        "landfill_bio_gas"=>114.8,
-        "propane"=>138.6,
-        "diesel_oil"=>163.1
+        "natural_gas"=>117.03,
+        "landfill_bio_gas"=>115.38,
+        "propane"=>139.16,
+        "diesel_oil"=>163.61
```

Comment:
@zolanaj Yes, the NOx cost equation only accounts for health-related costs that are a direct result of the emissions (rather than health outcomes from climate change). 

However, isn't CO2e a combination of CO2, CH4, and N2O (which is not a subset of NOx emissions?)?